### ISSUE TITLE
Reactivity warns for signals in addEventListener handlers

### ISSUE THREAD

**Issue by joshwilsonvu** [joshwilsonvu/eslint-plugin-solid#11]
**Describe the bug**
Event handlers are a special case in the `reactivity` rule where it's okay to read reactive variables even though the function isn't tracked by Solid. These are already accounted for in certain places:

```jsx
element.onclick = () => console.log(signal());

<div onClick={() => console.log(signal())} \>

<div on:click={() => console.log(signal())} />
```

But the following warns:

```js
element.addEventListener("click", () => console.log(signal()));
```

**To Reproduce**
Configure plugin, check last example. 

**Expected behavior**
Event handlers added with `addEventListener` should be treated as called functions.
